### ISSUE TITLE
Possible bug: dot-notation flagging boolean property?

### ISSUE THREAD

**Issue by helixbass** [helixbass/eslint-plugin-coffee#19]
This is getting flagged by `dot-notation`:
```
a.b[true]
```
But how should that be written in dot notation? Is this flagged by JS' `dot-notation`?

**Comment by helixbass** [comment]
I guess this is also flagged by JS' `dot-notation` so isn't a bug (rewrites to eg `a.b.true`)
